Update deployment to Cloud description (#45)
contember/docs: docs/guides/deploy-contember.md

---
title: Deploy Contember project to Contember Cloud
---

> This tutorial will guide you through the process of deploying Contember from local machine to Contember Cloud. (If you selfhost Contember, this tutorial won't work without admin server for which you need Enterprise license).

Reviewer: ```suggestion
This tutorial will guide you through the process of deploying the Contember from local machine to the Contember Cloud. (Tutorial won't work for self-hosted Contember without an admin server. The admin server is under an Enterprise license, [contact us](mailto:team@contember.com) for more information).
```

#### 1. Make sure you have correctly set project name

> Your project needs to have the same name locally and in Contember Cloud. Mostly (for example in our templates) there are 2 places where you need to set it:

Reviewer: ```suggestion
Your project needs to have the same name locally and in the Contember Cloud. In particular, there are 2 places where you need to set it up:
```

1. In `docker-compose.yml` set the `CONTEMBER_PROJECT_NAME` to your project slug
2. In `.env.production` set the `VITE_CONTEMBER_ADMIN_PROJECT_NAME` to your project slug

#### 2. Production build admin

Run build script inside your project folder:

```bash
npm run build-admin
```

#### 3. Deploy your project

> Now you can deploy your project using Contember CLI. You'll need deploy token (you'll get one when creating project in Contember Cloud administration).

Reviewer: ```suggestion
Now you can deploy your project using the Contember CLI. You'll need deploy token (you get one when creating a project in the Contember Cloud administration).
```
